test/example/qt.erl

```diff
@@ -202,6 +202,31 @@ client(Conns, Streams, CNo, SNo) ->
              end, lists:seq(CNo, CNo+N-1)),
       client(CC ++ Conns, Streams, CNo+N, SNo);
 
+    {ok, p_connect} ->
+      %% Use controlling process
+      Self = self(),
+      _P = proc_lib:spawn_link(
+            fun() ->
+                case quicer:connect("localhost", Port, c_opts(), 5000) of
+                  {ok, Conn} ->
+                    ok = quicer:controlling_process(Conn, Self),
+                    timer:sleep(400),
+                    Self ! {connection, Conn};
+                  Err ->
+                    Self ! {connection_err, Err}
```

Comment:
looks like a monitoring issue,
add
`timer:sleep(1000)` solved the issue.

The connection is closed because the old owner is dead.  I will check if it demon the old owner pid while swap the new owner.

Reply:
solution: https://github.com/emqx/quic/pull/172